plugins/woocommerce-admin/client/products/fills/more-menu-items/feedback-menu-item.tsx

```diff
@@ -33,10 +35,15 @@ export const FeedbackMenuItem = ( { onClose }: { onClose: () => void } ) => {
 							'woocommerce'
 						),
 					},
-					{ shouldShowComments: () => true },
+					{
+						shouldShowComments: () => true,
+					},
 					{
 						type: 'snackbar',
 						icon: <span>🌟</span>,
+					},
+					{
+						block_editor: isDescendantOf( 'product-block-editor' ),
```

Comment:
Any reason we don't continue to use the `context` property?  This seems like it might be an easier and more consistent way of checking context as we continue to iterate on the experience.

Reply:
@joshuatf By the use of the `context` property, do you mean assigning that to the `layoutString` value, or do you just mean using a property called `context` here that provides a string that indicates the placement? 

For instance, manually setting `{ context: 'product-block-editor' }` versus just setting it to the current value of `layoutString` (as we've done elsewhere). 

The reason I didn't set it equal to `layoutString` is that it's a bit more fragile, as I believe you brought up on the original layoutContext PR a while back. If someone adds another provider the string will change and potentially break that implementation. 

I'm receptive to making this something along of the lines of `context: 'product-block-editor'` as mentioned instead of a boolean value, although then I actually have to come up with a name for the non-block react product editor for it's context value 😢 .

Reply:
>The reason I didn't set it equal to layoutString is that it's a bit more fragile, as I believe you brought up on the original layoutContext PR a while back. If someone adds another provider the string will change and potentially break that implementation.

Ya, that's a good point.  I think what you have here is good, no reason to introduce complications to it for minimal benefits.